deploy/evs-csi-plugin/kubernetes/csi-evs-driver.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: storage.k8s.io/v1
+kind: CSIDriver
+metadata:
+  name: evs.csi.openstack.org
+spec:
+  attachRequired: true
+  podInfoOnMount: true
+  volumeLifecycleModes:
+    - Persistent
```

Comment:
pls add a black line

Reply:
done, has changed in branch "feat-documents-yaml"